sbi/src/sbi.rs

```diff
@@ -738,6 +738,35 @@ pub enum TeeFunction {
     ///
     /// a6 = 15
     TsmLocalFence,
+    /// Defines the single GPA range that will be used to map non-confidential shared pages.
+    /// The call made not be made after TVM is finalized, and the pages in the range cannot overlap
+    /// with the GPAs for confidential pages
+    /// a6 = 17
+    TsmDefineSharedPagesRange {
+        /// a0 = guest id
+        guest_id: u64,
+        /// a1 = guest physical base address of the shared range
+        guest_addr: u64,
+        /// a2 = page size (must be 4K for now)
+        page_type: TsmPageType,
```

Comment:
Do we care about the page type at this point? The range is unpopulated until the host goes to insert a page, at which point it can choose what page size to use.

Reply:
The intent was to able to compute the upper bound, but we can just pass it directly.